src/main/java/seedu/address/model/person/Date.java

```diff
@@ -48,4 +48,16 @@ public boolean equals(Object other) {
     public static boolean isValidDate(String date) {
         return date.matches(DATE_VALIDATION_REGEX);
     }
+
+    /**
+     * Returns true if a given string is a valid calendar date.
+     */
+    public static boolean isValidCalendarDate(String date) {
+        int days = Integer.parseInt(date.substring(0, 2));
+        int month = Integer.parseInt(date.substring(3, 5));
+        if (days > 28 && month == 2 ) {
```

Comment:
Have to check for leap years, it would be 29 days instead.

Reply:
done